lua/auto-dark-mode/init.lua

```diff
@@ -90,6 +95,9 @@ local function init()
 			"string:'org.freedesktop.appearance'",
 			"string:'color-scheme'",
 		}, " ")
+	elseif system == "WSL" then
+		query_command =
+			'reg.exe Query "HKCU\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Themes\\Personalize" /v AppsUseLightTheme | grep "AppsUseLightTheme"'
 	elseif system == "Windows_NT" then
 		-- Don't swap the quotes; it breaks the code
 		query_command =
```

Comment:
I wonder if the Windows_NT query would work for WSL as well. Does it have `findstr`?

Reply:
In both Windows_NT and WSL environments, the same commands can be used, but they need to be in lowercase and include the ".exe" extension.